pkg/operator/ceph/object/spec.go

```diff
@@ -76,6 +76,19 @@ func (c *clusterConfig) createDeployment(rgwConfig *rgwConfig) (*apps.Deployment
 			MaxSurge:       &intstr.IntOrString{IntVal: int32(0)},
 		}
 	}
+	hpas, err := c.context.Clientset.AutoscalingV1().HorizontalPodAutoscalers(c.store.Namespace).List(c.clusterInfo.Context, metav1.ListOptions{LabelSelector: c.storeLabelSelector()})
+	if err == nil {
+		if len(hpas.Items) > 0 {
+			for _, hpa := range hpas.Items {
```

Comment:
Secondly, I am not quite whether it is recommended to add multiple hpas for deployment. Technically it is possible, so I considered that scenarios

Reply:
What would that mean to have multiple hpas? Wouldn't they conflict to modify the deployment replicas?